src/playcanvas-anim.js

```diff
@@ -1519,10 +1914,16 @@ AnimationClip.prototype.setInterpolationType = function (type) {
     }
 };
 
-// *===============================================================================================================
-// * class AnimationEvent:
-// *===============================================================================================================
-var AnimationEvent = function AnimationEvent(name, time, fnCallback, context, parameter) {
+/**
+ * @constructor
+ * @param {string} name
+ * @param {number} time
+ * @param {AnimationEventCallback} fnCallback
+ * @param {any} context
+ * @param {any} parameter
+ */
+
+var AnimationEvent_ = function (name, time, fnCallback, context, parameter) {
```

Comment:
I don't like an underscore stuck on the end here to get around a name clash. Can you change it back for now until I decide what a good solution is? For now, it can just be a 'known issue'.